### PR TITLE
fix: post type dropdown position

### PR DESCRIPTION
**Pull request by darconny**

### CODE REVIEW

File: client/app/scripts/liveblog-edit/views/main.html [hunk 1]
```diff
@@ -217,7 +217,7 @@ <h3 class="page-nav-title">{{ blog.title }}</h3>
                             </div>
                         </div>
                     </div>
-                    <div class="freetype-selector" ng-click="$event.stopPropagation();" ng-if="selectPostTypeDialog">
+                    <div class="freetype-selector" ng-click="$event.stopPropagation();" ng-if="selectPostTypeDialog" ng-class="{'default': selectedPostType === 'Default'}">
```
Comment (m0g) [inline client/app/scripts/liveblog-edit/views/main.html]: Is it normal `'default': selectedPostType === 'Default'` is written with a capital D?

Comment (darconny) [inline client/app/scripts/liveblog-edit/views/main.html]: That's how it was in another tag, I suppose it was set in js like that.

Comment (m0g) [inline client/app/scripts/liveblog-edit/views/main.html]: Yeah you're right. That's why I approved the PR

Comment (darconny) [inline client/app/scripts/liveblog-edit/views/main.html]: OK I have to move the origin, for tests to pass.

**Pull request closed without merging**